books/free-programming-books-es.md

```diff
@@ -216,13 +216,17 @@
 
 #### AngularJS
 
-> :information_source: Véase también &#8230; [Angular](#angular)
-
 * [AngularJS](https://eladrodriguez.gitbooks.io/angularjs) - Elad Rodriguez (HTML)
+* [Angular 2](https://rldona.gitbooks.io/angular-2-book) - Raúl López (HTML)
+* [Aprendiendo Angular](https://ngchallenges.gitbook.io) - Vanessa Aristizabal (HTML)
+* [Aprendizaje Angular](https://riptutorial.com/Download/angular-es.pdf) - Compiled from StackOverflow Documentation (PDF)
+* [Aprendizaje Angular 2](https://riptutorial.com/Download/angular-2-es.pdf) - Compiled from StackOverflow Documentation (PDF)
+* [Entendiendo Angular](https://jorgeucano.gitbook.io/entendiendo-angular/) - Jorge Cano (HTML, [:package: ejemplos](https://github.com/jorgeucano/entendiendo-angular))
```

Comment:
one link only please

Reply:
Only left the main link to access the book was left.